src/mrpro/data/_kdata/KData.py

```diff
@@ -96,6 +124,17 @@ def from_file(
 
         acqinfo = AcqInfo.from_ismrmrd_acquisitions(acquisitions)
 
+        user_index = (acqinfo.idx.user5, acqinfo.idx.user6)
+
+        for indx in user_index:
+            if len(torch.unique(indx)) > 1:
```

Comment:
I would just write both checks explicitly:
If len(torch.unique(acqinfo.idx.user6) >1 or len(torch.unique(acqinfo.idx.user6) >1:


This is imho easier to see what is actually checked and it is only two cases. I don't see an advantage of a 'smarter' loop for only two special cases.